patches.txt

```diff
@@ -0,0 +1 @@
+bench.patches.v3.deprecate_old_config
```

Comment:
You might have created this accidentally.

Reply:
Deleted